doc/user/README.md

```diff
@@ -52,7 +52,7 @@ visually match.
 You can see how commonly rendered elements look by going to
 [`localhost:1313/stylesheet`](http://localhost:1313/stylesheet).
 
-You can use this as a scratch area by editing `content/stylesheet.md`.
+You can use this as a scratch area by editing [content/stylesheet.md](/doc/user/content/stylesheet.md).
```

Comment:
By the way, if you use relative links `(content/stylesheet.md)` vs `(/doc/user/content/stylesheet.md)` the links will be resilient to future movement of `doc/user`.